_includes/social/icons.md

```diff
@@ -86,9 +98,9 @@
 {% assign tb_on = site.data.social.tumblr or site.theme_settings.tumblr %}
 {% include social/icon_partial.html isDisplayed=tb_on url=tb_url social="Tumblr" %}
 
-{% assign gl_url = "https://gitlab.com/" | append: site.data.social.gitlab %}
-{% assign gl_on = site.data.social.gitlab or site.theme_settings.gitlab %}
-{% include social/icon_partial.html isDisplayed=gl_on url=gl_url social="Gitlab" %}
+{% assign tg_url = "https://t.me/@" | append: site.data.social.telegram %}
+{% assign tg_on = site.data.social.telegram or site.theme_settings.telegram %}
+{% include social/icon_partial.html isDisplayed=tg_on url=tg_url social="Telegram" %}
```

Comment:
Did you test that the icon exist for Telegram, Mastodon and Matrix?

Reply:
Oh ok just saw your comment in the PR.
If there are no icon for Matrix then it should not be in the there.

Reply:
Well I'd rather stay with Font Awesome for now.